session.go

```diff
@@ -160,3 +160,30 @@ func (s *session) sendCommand(cmdType command, cmd string) Packet {
 	results := s.readNBytes(int(r1))
 	return newPacket(int(rep), results)
 }
+
+func (s *session) sendvoidCommand(cmdType command, cmd string) error {
```

Comment:
if not perform r1<=0 check, will return an error from GetResultObject function due to content ==0.
<code>
	if len(p.content) == 0 {
		return nil, errors.New("Command failed for an unknown reason")
	}
</code>